src/Factory.php

```diff
@@ -2,7 +2,14 @@
 
 namespace Chromatic\OrangeDam;
 
-use Chromatic\OrangeDam\Endpoints\{AssetLink, DataTable, Endpoint, MediaFile, OAuth2, ObjectManagement, Search};
+use Chromatic\OrangeDam\Endpoints\{AssetFile,
```

Comment:
Fancy. I was not aware of this formatting/syntax option.

Reply:
Neither was I! My code editor made that change automatically, wnen I added the new import.